### PR TITLE
Pal 958 palisade service kafka errors

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

**Comment by dev930018** [comment]
Tweaks such that both palisade-service and filtered-resource-service report themselves healthy in k8s

File: charts/palisade-service/templates/deployment.yaml [hunk 1]
```diff
@@ -62,20 +62,27 @@ spec:
           {{- toYaml .Values.resources | nindent 10 }}
         env:
         - name: "LOADER_PATH"
-          value: {{ include "palisade-service.deployment.path" . }}
-        {{- $envVars := pluck .Values.global.deployment .Values.global.env | first | default .Values.global.env._default }}
-        {{- range $item := $envVars }}
+          value: {{ include "filtered-resource-service.deployment.path" . }}
+        - name: "SPRING_CONFIG_ADDITIONAL_LOCATION"
+          value: {{ include "filtered-resource-service.config.path" . }}
```
Comment (pd104923) [inline charts/palisade-service/templates/deployment.yaml]: `filtered-resource-service`, are you sure about that?
```suggestion
          value: {{ include "palisade-service.deployment.path" . }}
        - name: "SPRING_CONFIG_ADDITIONAL_LOCATION"
          value: {{ include "palisade-service.config.path" . }}
```

**Review by pd104923 — CHANGES_REQUESTED** [review]
One minor question.

**Review by pd104923 — APPROVED** [review]
All good 😄

**Review by developer01189998819991197253 — APPROVED** [review]

**Review by dev-958 — APPROVED** [review]

**Pull request merged**